crud/common.go

```diff
@@ -0,0 +1,42 @@
+package crud
+
+import (
+	"context"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+type Tuple = []interface{}
+type Object = map[string]interface{}
```

Comment:
Let's make a such change and write 1 example for it. This will allow on a client-side:

1. Use map[string]interface{} as an Object:
```Go
crud.MapObject(map[string]interface{}{
    // Map init.
})
```
2. Use custom types and do an encoding optimization like we did with options encoding.

```suggestion
// in msgpack.go:
// type customEncoder = msgpack.CustomEncoder
type Object = customEncoder

type MapObject = map[string]interface{}

func (o MapObject) EncodeMsgpack(enc *encoder) {
    enc.Encode(o)
}
```

Reply:
Sorry, I don't understand how it should be used.
1. Should it look like that?

```golang
crud.NewUpsertObjectRequest(spaceName).
			Object(crud.MapObject(map[string]interface{}{
                         // Map init.
                        })).
			Operations(operations).
			Opts(simpleOperationOpts),

...
// UpsertObjectRequest helps you to create request object to call
// `crud.upsert_object` for execution by a Connection.
type UpsertObjectRequest struct {
	spaceRequest
	object     Object
	operations []Operation
	opts       UpsertObjectOpts
}

// NewUpsertObjectRequest returns a new empty UpsertObjectRequest.
func NewUpsertObjectRequest(space string) *UpsertObjectRequest {
	req := new(UpsertObjectRequest)
	req.initImpl("crud.upsert_object")
	req.setSpace(space)
	req.object = MapObject{}
	req.operations = []Operation{}
	req.opts = UpsertObjectOpts{}
	return req
}
```

Reply:
Done.